docs/README.md

```diff
@@ -56,6 +56,7 @@ The VTEX Minicart is a block that displays a summary list of all items added by
 | `customPixelEventId` | `string`   | Store event ID responsible for triggering the `minicart.v2` to automatically open itself on the interface. | `undefined`    |
 | `customPixelEventName` | `string`   | Store event name responsible for triggering the `minicart.v2` to automatically open itself on the interface. Some examples are: `'addToCart'` and `'removeFromCart'`. Notice that using this prop will make the `minicart.v2` open in **every** event with the specified name if no `customPixelEventId` is specified. | `undefined`    |
 | `classes`         | `CustomCSSClasses` | Used to override default CSS handles. To better understand how this prop works, we recommend reading about it [here](https://github.com/vtex-apps/css-handles#usecustomclasses). Note that this is only useful if you're importing this block as a React component.                                      | `undefined`           |
+| `popupWithLink`          | `boolean`  | If the minicart is in `popup` variation and is `hover` mode, this prop transforms the minicart button in a link to the checkout when clicked  | `false`        |
```

Comment:
I don't think we need to create another prop for this behavior. How about instead introducing a new possible value for the `variation` prop?

Currently `variation` accepts two values: `popup` and `drawer`. How about adding a new value, such as `popupLink` that achieves the behavior you're implemented?

We tend to prefer this kind of solution because we're not increasing this block's API size :)

Reply:
Hello @victorhmp !

Requested adjustments made, could you validate again, please?